src/screens/styles/controllerScreenStyle.js

```diff
@@ -7,10 +7,13 @@ export default StyleSheet.create({
       flexDirection: 'column',
       justifyContent: 'space-between'
     },
+    slider: {
```

Comment:
I'm confused why we need an extra one for "slider", otherwise having sliders as 2, and patientFace as 1 makes sense to me.

Reply:
Theoretically we might want to style the sliders inside the slider container differently. We were having issues yesterday and this was one of the things I tried to fix it (I thought they were laying out wrong) and it didn't fix it, but I also know it doesn't break anything. I just left it in because I thought it might be useful in the future. If you think it should be gone I can take it out.